src/App.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import './index.css';
+import Book from './components/Book';
+import Navbar from './components/Navbar';
+import Categories from './components/Categories';
+
+function App() {
+  return (
+    <>
+      <Router>
+        <div>
+          <Navbar />
+          <div>
+            <Routes>
+              <Route path="/book" exact element={<Book />} />
```

Comment:
- [ ] Please modify this route path to '/' in order to display the `<Books />` list on the homepage. 
- [ ] Also, the component that should be rendered at the '/' route should be a `<BooksList>` or `<Books>` depending on what you choose to call it. Please note that this component will render individual `<Book>` components which take the book data as a prop and renders the title and the author.

Reply:
- [ ] Like the last code reviewer said kindly make the route page to the Booklist page. I should see this page when opening the server.
- [ ] Also ı should see add book form on this page. Kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/190864888-ccb143af-19ea-4044-8ee5-0692f19b6751.png)